geopandas/tests/test_geoseries.py

```diff
@@ -182,6 +188,35 @@ def test_transform(self):
         with pytest.raises(ValueError):
             self.landmarks.to_crs(crs=None, epsg=None)
 
+    def test_estimate_utm_crs__geographic(self):
+        if PYPROJ_LT_3:
+            with pytest.raises(RuntimeError, match=r"pyproj 3\+ required"):
+                self.landmarks.estimate_utm_crs()
+        else:
+            assert self.landmarks.estimate_utm_crs() == CRS("EPSG:32618")
+            assert self.landmarks.estimate_utm_crs("NAD83") == CRS("EPSG:26918")
+
+    def test_estimate_utm_crs__projected(self):
+        if PYPROJ_LT_3:
+            return
+        assert self.landmarks.to_crs("EPSG:3857").estimate_utm_crs() == CRS(
+            "EPSG:32618"
+        )
```

Comment:
```suggestion
    @pytest.mark.skipif(PYPROJ_LT_3, reason="requires pyproj 3 or higher")
    def test_estimate_utm_crs__projected(self):
        assert self.landmarks.to_crs("EPSG:3857").estimate_utm_crs() == CRS(
            "EPSG:32618"
        )
```

Can we rather skip this test for older pyproj? Same below.

Reply:
Good call. Will need to follow up later.